### DISCUSSION TITLE
Decentralized scss files

### DISCUSSION THREAD

**Discussion by HasinaNjaratin** [RyanClementsHax/storybook-addon-next#42] [General]
Is there a possibility (option) to use the add-on in case the scss files are in the relevant component folder but not centralized in a single 'styles' folder as in the example?

Here is my file structure:
```
src/
    components/
           Button/
                 Button.tsx
                 Button.stories.tsx
                 Button.module.scss
```
....

Use of Nextjs 12 too.

Thx

**Comment by RyanClementsHax** [comment]
Hello!

Thanks for the question. Does this currently not work for you?

**Reply by RyanClementsHax** [reply]
@HasinaNjaratin If this isn't working for you, then feel free to open up a bug report. I'd be happy to help you out! 

I will need a reproduction repo from you as I'm unable to replicate this locally (perhaps I'm not fully understanding what is going wrong).
